.evergreen/csfle/gcpkms/remote-scripts/startup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Delete the GCE instance after a period of time.
+# Refer: https://cloud.google.com/community/tutorials/create-a-self-deleting-virtual-machine
+sleep 7200
```

Comment:
Do we always leave the instance alive for 2 hours? Can we delete it sooner when the EVG task completes?

Reply:
I expect drivers to delete as soon as the task finishes with a `teardown_group`. Here is the [example](https://github.com/mongodb/mongo-go-driver/pull/1029/files#diff-2bc841e86ce96b7b422ae203fd8315d0b2a461956cecbe0e096420656fc3fb12R2280) with the Go implementation. The two hour limit is a fallback in case instances are leftover during development, or due to an unforeseen error.

Reply:
SGTM